getting-started/go.md

```diff
@@ -1 +1,112 @@
-TODO
+# Getting Started with IPLD in Go
+
+In Go, the primary way to use IPLD is via the `github.com/ipld/go-ipld-prime`
+module.
+
+Its API is centered around `Node`s, which can be built from scratch and encoded
+into blocks, where we can then obtain `Link`s to them.
+
+First, we'll need to import some IPLD packages and define a static link builder:
+
+```go
+package main
+
+import (
+	"github.com/ipfs/go-cid"
+	"github.com/ipld/go-ipld-prime"
+	_ "github.com/ipld/go-ipld-prime/codec/dagcbor"
+	"github.com/ipld/go-ipld-prime/fluent"
+	cidlink "github.com/ipld/go-ipld-prime/linking/cid"
+	basicnode "github.com/ipld/go-ipld-prime/node/basic"
+)
+
+var linkBuilder = cidlink.LinkBuilder{cid.Prefix{
+	Version:  1,    // Usually '1'.
+	Codec:    0x71, // dag-cbor as per multicodec
+	MhType:   0x15, // sha3-384 as per multicodec
+	MhLength: 48,   // sha3-224 hash has a 48-byte sum.
+}}
+```
+
+With that out of the way, we can build our first nodes. Beware that, for the
+sake of brevity, we aren't handling any errors.
```

Comment:
ymmv/nit: for myself, I like [sentence wrapping](https://twitter.com/garybernhardt/status/1167123539834523648).

(I don't care one way or the other at all.  But I definitely don't constrain myself to 80-chars, or any hard-wrap rule at all that would cause reflows.  So feel equally free :))

Reply:
I don't have strong feelings here. But I think that, for short paragraphs, wrapping is fine.